client/signup/steps/site-topic/index.jsx

```diff
@@ -64,15 +62,11 @@ class SiteTopicStep extends Component {
 
 	onSubmit = event => {
 		event.preventDefault();
-
-		this.props.submitSiteTopic( this.trimedSiteTopicValue() );
+		this.props.submitSiteTopic( this.state.siteTopicValue );
 	};
 
-	trimedSiteTopicValue = () => this.state.siteTopicValue.trim();
```

Comment:
By way of explanation, I removed this because `SiteVerticalsSuggestionSearch` will trim the value out of the box.